frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -42,4 +44,33 @@ describe('Localization', () => {
     cy.visit('/dashboards?pseudolocalization=true&lng=en');
     cy.byTestID('utilization-card-item-text').isPseudoLocalized();
   });
+
+  it('pseudolocalizes monitoring pages', () => {
+    const monitoringPages = ['alerts', 'silences', 'alertrules'];
+    monitoringPages.forEach((page) => {
+      cy.visit(`/monitoring/${page}?pseudolocalization=true&lng=en`);
+      if (page === 'alerts' || page === 'alertrules') {
+        listPage.rows.shouldBeLoaded();
+      }
+      // check applied filters are i18ned then clear all filters
+      cy.get('.pf-c-chip-group__main').then((groups) => {
+        if (groups.length >= 1) {
+          cy.get('[class*="pf-c-chip-group"]').isPseudoLocalized();
+        }
+        listPage.filter.clearAllFilters();
+      });
+      cy.get(ListPageSelector.tableColumnHeaders).isPseudoLocalized();
+      listPage.filter.clickFilterDropdown();
+      cy.get('.co-filter-dropdown-item__name').isPseudoLocalized();
+      if (page === 'alerts' || page === 'alertrules') {
+        if (page === 'alerts') {
+          cy.get(DetailsPageSelector.horizontalNavTabs).isPseudoLocalized();
+        }
+        listPage.filter.clickSearchByDropdown();
+        cy.get('[data-test-id="dropdown-menu"]').isPseudoLocalized();
+      } else {
+        cy.get('.pf-c-button.pf-m-primary').isPseudoLocalized();
```

Comment:
I'm wondering if you should just add i18n checks to [monitoring.spec.ts](https://github.com/openshift/console/blob/68f58f20c2ad43b0cab33d5a4719900ff17caf44/frontend/packages/integration-tests-cypress/tests/monitoring/monitoring.spec.ts#L59-L58) where we are already loading Alerts and Silences and are doing `testA11y` testing.   It seems like the only i18n test you are doing for `alertrules` is testing this one button -which you could add to `monitoring.spec.ts`.